<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1539</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcome">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>-10</y>
      <width>881</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 55pt &quot;Algerian&quot;;
color:brown;</string>
    </property>
    <property name="text">
     <string>WELCOME TO LIBRO</string>
    </property>
   </widget>
   <widget class="QLabel" name="pics">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>511</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> text-align: center;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/pictures/intro.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="log_in">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_admin">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>280</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>sign as admin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_in">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>sign In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_admin">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>350</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>log as admin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>741</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pictures/read.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="read">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>600</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Sitka&quot;;
font: 700 9pt &quot;Bahnschrift&quot;;
color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>READ CAPTION</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1539</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
